src/main/resources/application-it.properties

```diff
@@ -0,0 +1,43 @@
+spring.main.banner-mode=off
+
+# Scheduling
+cron.expression=0 0 0 31 FEB ?
```

Comment:
Instead of setting an invalid date here, why not disable it instead?
(A hyphen disables the cron expression)